Elevel/Elevel.Api/Controllers/TestController.cs

```diff
@@ -28,12 +30,16 @@ public async Task<IActionResult> CreateTestAsync ([FromBody] CreateTestCommand.R
         [Authorize(Roles = nameof(UserRole.HumanResourceManager)),HttpPost("assign")]
         public async Task<IActionResult> AssignTestAsync([FromBody] AssignTestCommand.Request request)
         {
+            var claims = User.Claims.ToList();
+            var userId = claims.FirstOrDefault(x => x.Type == "uid").Value;
+            request.HrId = Guid.Parse(userId);
+
             var result = await Mediator.Send(request);
             return Ok(result);
         }
 
-        [HttpGet("{id:Guid}")]
-        public async Task<IActionResult> GetTestByIdAsync([FromRoute] GetTestByIdQuery.Request request)
+        [HttpPut("startTest/{id:Guid}")]
```

Comment:
it's better to change Url to Id/start
because right now it looks like api/**tests**/start**Test**/Id
and I suggest api/tests/id/start